### PR TITLE
Fix contributing broken link and config PR template

### PR DESCRIPTION
**Pull request by VaneskaSousa**
- Fix issue #36 
- Moving PR template for .github to config this by develop branch default 
- Create a custom issue template for issues which don't fit as a bug report or issue request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/custom.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Custom issue
+about: Use this template just if your issue don't fit as a bug report or a issue template
```
Comment (nmarquesin) [inline .github/ISSUE_TEMPLATE/custom.md:3]: Please change: 
- "don't fit" to "doesn't fit" 
- "a issue template" to "an issue template"

File: .github/ISSUE_TEMPLATE/custom.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Issue Title
+A clear and objective resume of your needs.
```
Comment (nmarquesin) [inline .github/ISSUE_TEMPLATE/custom.md:11]: Please change:
- "resume" to "summary"

File: .github/ISSUE_TEMPLATE/custom.md (hunk 1 above)
Comment (VaneskaSousa) [reply .github/ISSUE_TEMPLATE/custom.md:3]: ow, "issue template" was a little mistake. the correct is a feature report

File: .github/ISSUE_TEMPLATE/custom.md (hunk 2 above)
Comment (VaneskaSousa) [reply .github/ISSUE_TEMPLATE/custom.md]: done

File: .github/ISSUE_TEMPLATE/custom.md (hunk 1 above)
Comment (VaneskaSousa) [reply .github/ISSUE_TEMPLATE/custom.md]: done

**Pull request merged**